contracts/MarginBase.sol

```diff
@@ -262,6 +262,11 @@ contract MarginBase is MinimalProxyable, IMarginBase, OpsReady {
         notZero(_amount, "_amount")
         onlyOwner
     {
+        _deposit(_amount);
+    }
+
+    /// @dev see deposit() NatSpec
+    function _deposit(uint256 _amount) internal {
```

Comment:
Because this doesn't have the notZero check you can bypass it with `depositAndDistribute`

Reply:
Fixed